rush.json

```diff
@@ -606,6 +606,14 @@
       "shouldPublish": false
     },
 
+    {
+      "packageName": "@kadena-dev/lint-package",
+      "tags": ["tools"],
+      "projectFolder": "packages/tools/lint-package",
+      "shouldPublish": false,
+      "versionPolicyName": "alpha"
```

Comment:
We must not have a `versionPolicyName` when `shouldPublish=false`.

(This should be part of the linter maybe?)

Reply:
Alright, thanks